File: python/src/tests/t_init_shm_log.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

count = int(sys.argv[1])

print('init shm log : ' + str(count) + ' : ')

area = atrshmlog.get_area()

iret = atrshmlog.init_shm_log(area, count)

print('init shm log : ' + str(iret) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_init_shm_log.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$count = shift @ARGV;

print "init shm log : $count : \n";

$area = Atrshmlog::get_area();

$iret = Atrshmlog::init_shm_log($area, $count);

print "init shm log : $iret : \n";


print " \n";

exit (0);

# end of main
